_posts/2017-03-30-fact-skill-post.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title: My First Alexa Fact Skill
+feature-img: "img/min-back.jpg"
+---
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;During my Bloc Apprenticeship, I moved on to learning another skill Alexa could utilize. It was even easier than when I started learning about making a trivia skill.  This time I was learning on how to make a fact skill for Alexa. Again, everything was well documented step by step. I ended up writing a fact skill about technology called Tech Facts.  The hardest part about it was looking up facts that I thought were interesting. The part I found interesting was how the code was updated. When learning the Trivia skill, we used a blank slate and uploaded the code in a .zip folder. With the fact skill I chose the fact template, which had a preloaded library and had to copy and paste the index file into their code editor. I suggest you take a look at their [github](https://github.com/alexa/skill-sample-nodejs-fact) repoand start building a skill of your own.
```

Comment:
Start off the post with two of your quiz questions/answers and ask the reader if they can guess what your Fact based quiz app is based on. 

Something like: "Alexa, --question". Alexa: "answer". Have fun with it!

"The part I found interesting" should be a new paragraph. 

"With the fact skill I chose the fact template" => "I chose the fact template,"

"repoand" => "repository to start building a skill of your own"

Lastly, did you use a reputable source for the facts?